lib/takeaway.rb

```diff
@@ -0,0 +1,40 @@
+# require_relative 'send_sms'
```

Comment:
Maybe the takeaway file and spec could have been divided into separate classes, such as order or menu as well. Apart from that, this is very readable and easy to follow, well done!